modules/navier_stokes/src/actions/NSFVAction.C

```diff
@@ -388,6 +388,13 @@ NSFVAction::validParams()
       "The numerical scheme to interpolate the passive scalar field variables to the "
       "face (separate from the advected quantity interpolation).");
 
+  MooseEnum velocity_interpolation("average rc", "rc");
```

Comment:
I thought we were trying to avoid giving that average option so we did not have to support users getting a checkerboard and complaining

Reply:
The default is `rc`. If they try something other than the default, then presumably they are already having problems. I think it would be nice to be able to do anything with the action that you can do with the individual objects. Moreover, for buoyant flow for high Rayleigh numbers, I have seen that the only way you can get good results with FV matching FE (at least sans turbulence models) is to use a TVD limiter with average velocity interpolation. There is no checkerboard in this case (or at least I haven't seen it yet on Sinan's cask problems).